templates/csi-daemonset.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if and (eq (.Values.csi.enabled | toString) "true" ) (eq (.Values.global.enabled | toString) "true") }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
```

Comment:
We should add the ability to apply annotations to these pods like we do for our other deployments.